File: weeks/week_1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user value
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // printvalue


    int i = 0;
    int j = 0;
    int k = height;
    int l = height;

    while (i < height) // espace pyramid
    {
        j = j + 1;
        while (j < height) // print .
        {
            printf(" ");
            j++;
        }

        while (j <= k)
        {
            printf("#");
            j++;
        }
        printf("  ");
        while (k + 1 >= j)
        {
            printf("#");
            j++;
        }
        printf("\n");
        height--;
        j = 0;

    }
}

File: weeks/week_6/sentimental-mario-more/mario.py
import cs50


def main():
    # prompt user for a number between 1 and 8 using get int
    while True:
        height = cs50.get_int("Entrer une hauteur : ")
        if height >= 1 and height <= 8:
            break

    # looping process
    for i in range(1, height + 1):
        print(" " * (height - i) + "#" * i + "  " + "#" * i)


main()